### ISSUE TITLE
undefined reference to `gtk_widget_insert_action_group' error

### ISSUE THREAD

**Issue by HiTuX** [lanoxx/tilda#210]
I'm trying to compile the latest release on ubuntu12.04, using `./autogen.sh --enable-vte-2.91=no` I got this error msg:
```
$ make --silent
Making all in po
/usr/bin/msgmerge --update  --lang=bg bg.po tilda.pot
............................. done.
rm -f bg.gmo && /usr/bin/msgfmt -c --statistics --verbose -o bg.gmo bg.po
bg.po: 89 translated messages, 47 fuzzy translations, 93 untranslated messages.
/usr/bin/msgmerge --update  --lang=ca ca.po tilda.pot
........................... done.
rm -f ca.gmo && /usr/bin/msgfmt -c --statistics --verbose -o ca.gmo ca.po
ca.po: 33 translated messages, 13 fuzzy translations, 183 untranslated messages.
/usr/bin/msgmerge --update  --lang=cs cs.po tilda.pot
............................. done.
rm -f cs.gmo && /usr/bin/msgfmt -c --statistics --verbose -o cs.gmo cs.po
cs.po: 89 translated messages, 46 fuzzy translations, 94 untranslated messages.
/usr/bin/msgmerge --update  --lang=de de.po tilda.pot
............................... done.
rm -f de.gmo && /usr/bin/msgfmt -c --statistics --verbose -o de.gmo de.po
de.po: 174 translated messages, 10 fuzzy translations, 45 untranslated messages.
/usr/bin/msgmerge --update  --lang=el el.po tilda.pot
................................ done.
rm -f el.gmo && /usr/bin/msgfmt -c --statistics --verbose -o el.gmo el.po
el.po: 2 translated messages, 7 fuzzy translations, 220 untranslated messages.
/usr/bin/msgmerge --update  --lang=en_GB en_GB.po tilda.pot
........................... done.
rm -f en_GB.gmo && /usr/bin/msgfmt -c --statistics --verbose -o en_GB.gmo en_GB.po
en_GB.po: 82 translated messages, 44 fuzzy translations, 103 untranslated messages.
/usr/bin/msgmerge --update  --lang=es es.po tilda.pot
............................ done.
rm -f es.gmo && /usr/bin/msgfmt -c --statistics --verbose -o es.gmo es.po
es.po: 181 translated messages, 8 fuzzy translations, 40 untranslated messages.
/usr/bin/msgmerge --update  --lang=fr fr.po tilda.pot
............................ done.
rm -f fr.gmo && /usr/bin/msgfmt -c --statistics --verbose -o fr.gmo fr.po
fr.po: 108 translated messages, 39 fuzzy translations, 82 untranslated messages.
/usr/bin/msgmerge --update  --lang=hu hu.po tilda.pot
.............................. done.
rm -f hu.gmo && /usr/bin/msgfmt -c --statistics --verbose -o hu.gmo hu.po
hu.po: 90 translated messages, 47 fuzzy translations, 92 untranslated messages.
/usr/bin/msgmerge --update  --lang=it it.po tilda.pot
.............................. done.
rm -f it.gmo && /usr/bin/msgfmt -c --statistics --verbose -o it.gmo it.po
it.po: 142 translated messages, 23 fuzzy translations, 64 untranslated messages.
/usr/bin/msgmerge --update  --lang=lt lt.po tilda.pot
.............................. done.
rm -f lt.gmo && /usr/bin/msgfmt -c --statistics --verbose -o lt.gmo lt.po
lt.po: 187 translated messages, 8 fuzzy translations, 34 untranslated messages.
/usr/bin/msgmerge --update  --lang=pl pl.po tilda.pot
........................... done.
rm -f pl.gmo && /usr/bin/msgfmt -c --statistics --verbose -o pl.gmo pl.po
pl.po: 75 translated messages, 55 fuzzy translations, 99 untranslated messages.
/usr/bin/msgmerge --update  --lang=pt pt.po tilda.pot
............................ done.
rm -f pt.gmo && /usr/bin/msgfmt -c --statistics --verbose -o pt.gmo pt.po
pt.po: 90 translated messages, 47 fuzzy translations, 92 untranslated messages.
/usr/bin/msgmerge --update  --lang=pt_BR pt_BR.po tilda.pot
........................ done.
rm -f pt_BR.gmo && /usr/bin/msgfmt -c --statistics --verbose -o pt_BR.gmo pt_BR.po
pt_BR.po:7: header field 'Language-Team' missing in header
pt_BR.po:7: header field 'MIME-Version' missing in header
pt_BR.po: 121 translated messages, 2 fuzzy translations, 106 untranslated messages.
/usr/bin/msgmerge --update  --lang=ru ru.po tilda.pot
................................. done.
rm -f ru.gmo && /usr/bin/msgfmt -c --statistics --verbose -o ru.gmo ru.po
ru.po: 90 translated messages, 47 fuzzy translations, 92 untranslated messages.
/usr/bin/msgmerge --update  --lang=sk sk.po tilda.pot
......................... done.
rm -f sk.gmo && /usr/bin/msgfmt -c --statistics --verbose -o sk.gmo sk.po
sk.po: 56 translated messages, 15 fuzzy translations, 158 untranslated messages.
/usr/bin/msgmerge --update  --lang=sl sl.po tilda.pot
........................ done.
rm -f sl.gmo && /usr/bin/msgfmt -c --statistics --verbose -o sl.gmo sl.po
sl.po: 32 translated messages, 14 fuzzy translations, 183 untranslated messages.
/usr/bin/msgmerge --update  --lang=sv sv.po tilda.pot
......................... done.
rm -f sv.gmo && /usr/bin/msgfmt -c --statistics --verbose -o sv.gmo sv.po
sv.po: 56 translated messages, 15 fuzzy translations, 158 untranslated messages.
/usr/bin/msgmerge --update  --lang=tr tr.po tilda.pot
......................... done.
rm -f tr.gmo && /usr/bin/msgfmt -c --statistics --verbose -o tr.gmo tr.po
tr.po: 0 translated messages, 229 untranslated messages.
/usr/bin/msgmerge --update  --lang=zh_CN zh_CN.po tilda.pot
............................ done.
rm -f zh_CN.gmo && /usr/bin/msgfmt -c --statistics --verbose -o zh_CN.gmo zh_CN.po
zh_CN.po: 90 translated messages, 47 fuzzy translations, 92 untranslated messages.
touch stamp-po
  CC       src/src_tilda-callback_func.o
  CC       src/src_tilda-configsys.o
  CC       src/src_tilda-eggaccelerators.o
  CC       src/src_tilda-key_grabber.o
  CC       src/src_tilda-tilda.o
  CC       src/src_tilda-tilda_terminal.o
src/tilda_terminal.c: In function ‘menu_searchbar_cb’:
src/tilda_terminal.c:871:5: warning: implicit declaration of function ‘toggle_searchbar_cb’ [-Wimplicit-function-declaration]
src/tilda_terminal.c: In function ‘popup_menu’:
src/tilda_terminal.c:983:5: warning: implicit declaration of function ‘gtk_widget_insert_action_group’ [-Wimplicit-function-declaration]
  CC       src/src_tilda-tilda_window.o
src/tilda_window.c: In function ‘tilda_window_search_forward_cb’:
src/tilda_window.c:761:5: warning: passing argument 1 of ‘tilda_window_search’ from incompatible pointer type [enabled by default]
src/tilda_window.c:706:13: note: expected ‘struct GtkWidget *’ but argument is of type ‘struct GtkButton *’
src/tilda_window.c: In function ‘tilda_window_search_backward_cb’:
src/tilda_window.c:765:5: warning: passing argument 1 of ‘tilda_window_search’ from incompatible pointer type [enabled by default]
src/tilda_window.c:706:13: note: expected ‘struct GtkWidget *’ but argument is of type ‘struct GtkButton *’
  CC       src/src_tilda-tomboykeybinder.o
  CC       src/src_tilda-wizard.o
  CC       src/src_tilda-xerror.o
  GEN      src/glade-resources.c
  CC       src/src_tilda-glade-resources.o
  CCLD     src/tilda
src/src_tilda-tilda_terminal.o: In function `popup_menu':
tilda_terminal.c:(.text+0x22b2): undefined reference to `gtk_widget_insert_action_group'
collect2: ld returned 1 exit status
make[2]: *** [src/tilda] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

**Comment by lanoxx** [comment]
Hi and apologies for not replying earlier. The function in question was added with GTK+ 3.6, but on 12.04 (Codename: Precise Pangolin) there is only 3.4.2 available. Therefore you will not be able to build this on your system. Unfortunately it looks like I forgot to bump the version dependencies for GTK+ in the build system so it does not throw an error during configure but goes all the way though compilation and then fails during the linking phase.

I usually only try to support the most recent LTS release which is 14.04 in this case, so I recommend you try to use an older release of tilda.

**Comment by lanoxx** [comment]
I pushed 4847738c714ed51cd093b82bd17c84daf39b74ab for the tilda-1-3 branch and 4847738c714ed51cd093b82bd17c84daf39b74ab for the master branch, so tilda will not require at least GTK+ 3.10.0 (which is available on Ubuntu 14.04 LTS)
